### ISSUE TITLE
Monterey

### ISSUE THREAD

**Issue by xjwan110** [reynan2020/Asus-Laptop-14-X409UA---Opencore-EFI#1]
Have you been upgraded to Monterey yet?

**Comment by reynan2020** [comment]
Yes. Thanks for reminding me to update this repo.  I have not updated it for a while. I will upload the files later.

**Comment by xjwan110** [comment]
OK, thank you very much.

**Comment by xjwan110** [comment]
After you upgrade, does the computer sleep wake up normally？

**Comment by reynan2020** [comment]
No, sleep doesn't work sometimes.
